.github/actions/spelling/advice.md

```diff
@@ -1,27 +1,25 @@
 <!-- See https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples%3A-advice --> <!-- markdownlint-disable MD033 MD041 -->
-<details><summary>If you see a bunch of garbage</summary>
+<details><summary>If the flagged items are :exploding_head: false positives</summary>
```

Comment:
This is part of the refreshed workflow. You of course don't need to take the changes, but in speaking w/ some real life users, the changes here made for easier to understand messaging.

If you have feedback on the content, I welcome it, as the goal is to make every aspect of this tool as user-friendly as possible.